x/oracle/keeper/historic_price.go

```diff
@@ -47,55 +47,91 @@ func medianDeviation(median sdk.Dec, prices []sdk.Dec) sdk.Dec {
 	return medianDeviation
 }
 
-// GetMedian returns a given denom's median price in the last prune
-// period since a given block.
-func (k Keeper) GetMedian(
+// average returns the average value of a list of prices
```

Comment:
What do you think about moving all these functions that don't interact with the keeper somewhere else? They are mostly just calculations done on a set of prices, maybe something like `price_calculations.go`

Reply:
Added them in our utils package in pricestats.go 👍